avm/res/document-db/database-account/mongodb-database/collection/main.bicep

```diff
@@ -0,0 +1,53 @@
+metadata name = 'DocumentDB Database Account MongoDB Database Collections'
+metadata description = 'This module deploys a MongoDB Database Collection.'
+metadata owner = 'Azure/module-maintainers'
+
+@description('Conditional. The name of the parent Cosmos DB database account. Required if the template is used in a standalone deployment.')
+param databaseAccountName string
+
+@description('Conditional. The name of the parent mongodb database. Required if the template is used in a standalone deployment.')
+param mongodbDatabaseName string
+
+@description('Required. Name of the collection.')
+param name string
+
+@description('Optional. Name of the mongodb database.')
+param throughput int = 400
+
+@description('Required. Indexes for the collection.')
+param indexes array
+
+@description('Required. ShardKey for the collection.')
+param shardKey object
+
+resource databaseAccount 'Microsoft.DocumentDB/databaseAccounts@2023-04-15' existing = {
+  name: databaseAccountName
+
+  resource mongodbDatabase 'mongodbDatabases@2023-04-15' existing = {
+    name: mongodbDatabaseName
+  }
+}
+
+resource collection 'Microsoft.DocumentDB/databaseAccounts/mongodbDatabases/collections@2023-04-15' = {
+  name: name
+  parent: databaseAccount::mongodbDatabase
+  properties: {
+    options: contains(databaseAccount.properties.capabilities, { name: 'EnableServerless' }) ? null : {
+      throughput: throughput
+    }
+    resource: {
+      id: name
+      indexes: indexes
+      shardKey: shardKey
+    }
+  }
+}
+
+@description('The name of the mongodb database.')
```

Comment:
This and the other descriptions are incorrectly referring to. MongoDB

Reply:
Fixed: https://github.com/Azure/bicep-registry-modules/pull/667/commits/0e85651de73c0c7053a3cd8e6bd152ba5ab374d1